### ISSUE TITLE
WorkValidationException thrown by Gradle with 1.5.20-1.0.0-beta03

### ISSUE THREAD

**Issue by ZacSweers** [google/ksp#494]
When trying to update, I see this issue reported by Gradle

Repro: 
* https://github.com/ZacSweers/MoshiX/pull/128
* https://github.com/ZacSweers/MoshiX/pull/128/checks?check_run_id=2921448001

```
* What went wrong:
A problem was found with the configuration of task ':moshi-ksp:moshi-ksp:kspKotlin' (type 'KspTaskJvm').
  - In plugin 'com.google.devtools.ksp' type 'com.google.devtools.ksp.gradle.KspTaskJvm' method 'configure()' should not be annotated with: @Internal.
    
    Reason: Input/Output annotations are ignored if they are placed on something else than a getter.
> Task :moshi-ksp:moshi-ksp:kspKotlin FAILED
    
    Possible solutions:
      1. Remove the annotations.
      2. Rename the method.
    
    Please refer to https://docs.gradle.org/7.1/userguide/validation_problems.html#ignored_annotations_on_method for more details about this problem.

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

**Comment by RBusarow** [comment]
Dupe of #487 which was fixed (but not yet released) by #488.

**Comment by ting-yuan** [comment]
Sorry for the inconvenience. The fix just released: https://github.com/google/ksp/releases/tag/1.5.20-1.0.0-beta04
